src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/ExtendedQueryTag.sql

```diff
@@ -6,6 +6,7 @@
     TagPrivateCreator is identification code of private tag implementer, only apply to private tag.
     TagStatus can be 0, 1 or 2 to represent Adding, Ready or Deleting.
     QueryStatus can be 0, or 1 to represent Disabled or Enabled.
+    ResourceType = 0 means Image SOP instance, ResourceType = 1 means UPS SOP instance.
 **************************************************************/
 CREATE TABLE dbo.ExtendedQueryTag (
```

Comment:
We discuss this decision [here](https://dev.azure.com/microsofthealth/Health/_git/health-paas-docs?path=/specs/DICOM/UPS-RS/ups-rs.md&version=GBusers/kabalas/upsrs&_a=preview&anchor=storage) - the short version is that using the ExtendedQueryTag tables gives us the ability to search sequences without having to duplicate tables and stored procedures.

Reply:
We need to support sequencing, we could have created our own table to workitem query tags but we avoided since we can reuse the extended query tag itself.